### DISCUSSION TITLE
Disable freecom time & date prompt at startup

### DISCUSSION THREAD

**Discussion by haenschen** [dosemu2/dosemu2#2034] [Q&A]
Hi,

latest dosemu from devel with comcom32/fdpp/install-freedos from ppa afterwards.
I want to use freedos comcom (http://www.ibiblio.org/pub/micro/pc-stuff/freedos/files/repositories/1.3/pkg-html/freecom.html) which I placed in C:

On dosemu startup it always prompts me:
```
Current date is Thu 29.06.2023
Enter new date (dd.mm.[cc]yy):
Curent time is 19:35:46,30
Enter new time:
```

How to disable this?

**Comment by stsp** [comment]
By not using freecom?

**Reply by ecm-pushbx** [reply]
For some reason this comment didn't show up in my mail inbox and there is a different comment by haenschen which is in my inbox but not visible here. Odd.

**Reply by ecm-pushbx** [reply]
Your comment did arrive in my inbox after all.

**Reply by andrewbird** [reply]
The same wierdness happened to me also.

**Comment by andrewbird** [comment]
Or add an autoexec.bat file to the C: drive, you should be able to find fdautoem.bat in the dosemu source directory, or somewhere in the package install, on ubuntu it's at `/usr/share/dosemu/dosemu2-cmds-0.3/fdautoem.bat`. This should make a good basis for your custom autoexec.bat.

**Reply by haenschen** [reply]
Thanks!!

**Comment by haenschen** [comment]
What's the best practice to make use of freecom now? If I copy the command.com to drive_c I don't get the time/date prompt anymore, but it seems like the PATH variables are all wrong. I did `insfdusr` but e.g. xcopy isn't found while it works with comcom32...

**Reply by stsp** [reply]
So if you type `path` is it wrong?

**Reply by haenschen** [reply]
For comcom32 it is
```
path
PATH=D:\dosemu;E:\;F:\;C:\bin
```
For freecom
```
path
PATH=D:\dosemu;C:\;F:\;C:\bin
```
Previously copying command.com to drive_c and setting up an autoexec.bat as described by Andrew was enough :/

**Reply by stsp** [reply]
Path is correct.
`E:\` is needed because comcom32
is on `E:\`, and `C:\` is needed because
freecom is on `C:\`.
`xcopy` is simply not installed by
`insfdusr` because it is built into
comcom32/64.

**Reply by andrewbird** [reply]
If we were to add xcopy to the install script, would a comcom installation use its internal xcopy in preference?

**Reply by stsp** [reply]
I think so.
IIRC it prefers the built-ins.

**Reply by haenschen** [reply]
Well then, until Andrew adds xcopy I should probably copy all .exe utilities from freedos to C:\BIN :)
And to add onto this: comcom will prefer the built-ins as I experienced here: https://github.com/dosemu2/comcom64/issues/61

**Reply by stsp** [reply]
Andrew, would you like to instead
see why tests are failing with comcom64(32)?
This "lets make freecom better" can wait.
